agent/src/main/java/eu/cqse/teamscale/jacoco/util/LoggingUtils.java

```diff
@@ -53,9 +74,7 @@ public static void initializeLogging(Path loggingConfigFile) throws IOException
 			return;
 		}
 
-		ConfigurationSource source = new ConfigurationSource(new FileInputStream(loggingConfigFile.toFile()),
-				loggingConfigFile.toFile());
-		Configurator.initialize(null, source);
+		reconfigureLoggerContext(new FileInputStream(loggingConfigFile.toFile()));
 	}
 
 	/** Wraps the given log4j logger into an {@link ILogger}. */
```

Comment:
logback logger

Reply:
i wrote slf4j since it's technically the slf4j interface